vertical-pod-autoscaler/e2e/updater.go

```diff
@@ -78,18 +74,96 @@ var _ = UpdaterE2eDescribe("Updater", func() {
 				UpperBound:    resourceList,
 			}},
 		}
-
-		vpaClientSet := vpa_clientset.NewForConfigOrDie(config)
-		vpaClient := vpaClientSet.PocV1alpha1()
-		_, err = vpaClient.VerticalPodAutoscalers(ns).Create(vpaCRD)
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		InstallVPA(f, vpaCRD)
 
 		ginkgo.By("Waiting for pods to be restarted")
 
 		err = waitForPodSetChangedInDeployment(c, d, podList)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 	})
+
+	ginkgo.It("observes pod disruption budget", func() {
+
+		ginkgo.By("Setting up a hamster deployment")
+		c := f.ClientSet
+		ns := f.Namespace.Name
+
+		cpuQuantity := ParseQuantityOrDie("10m")
```

Comment:
Could you please extract this (up to podList creation) to a helper method? This code is very similar to the other test.